lib/utils/parseJson.js

```diff
@@ -1,11 +1,5 @@
-const { parse } = require('jju/lib/parse');
-
 const parseJson = (json, revivew) => {
-  try {
-    return parse(json, revivew);
-  } catch (error) {
-    throw error;
-  }
+  return JSON.parse(json, revivew);
```

Comment:
I'd add a comment here so it's more apparent why this is a separate module:

```js
/**
 * Parses JSON
 *
 * Previously we've been using jsonlint and jju for parsing. While we've stepped
 * back to use plain JSON.parse(), we keep our doors open to switch the parser
 * - which is the reason why this is abstracted into parseJson()
 */
const parseJSON = ...
```

Reply:
Added in 3aade08.